plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties

```diff
@@ -3,6 +3,7 @@ google.preferences=Google Cloud Platform-related settings
 
 # Standard deploy settings
 auto.promote=Promote the deployed version to receive all traffic
+deploy.config.files=Deploy config files together
```

Comment:
"config files" is not really terminology that is used for these. We need a better term. Let me think about that.